css-display-3/display-contents-tr-001-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reftest Reference</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<p>You should see the word PASS below.</p>
+<table cellpadding="0" cellspacing="0">
+    <td>P<span>A</span></td>
+    <td>S<span>S</span></td>
+</table>
```

Comment:
Regarding subject: are you sure this is about text shaping and not subtleties in size/position for table layout vs block layout?

I don't know what the guidelines say about implicit elements like tr, but I think it would be less confusing if you included the tr in the ref markup.